app/views/crosswalk_issues/find_matches.html.erb

```diff
@@ -41,7 +41,7 @@
                 <td><%= iped["cross"] %></td>
                 <td><%= (iped["match_score"].to_f * 100).round(3) %>%</td>
                 <td>
-                  <%= form_with(url: "/crosswalk_issues/match_ipeds_hd", method: "post") do %>
+                  <%= form_with(url: crosswalk_issues_match_ipeds_hd_path) do %>
```

Comment:
I think the routing file should match up with how the route is being used.  A request spec for `get crosswalk_issues_match_ipeds_hd_path` would raise an exception

Reply:
I'm not sure I'm following. I believe we are treating this as a `POST`, which is the default for `form_with`. I'm going to use you as a reference here:
https://github.com/department-of-veterans-affairs/gibct-data-service/pull/567#discussion_r377742119

I accidentally added back in the method when I was working on these new changes. I went ahead and changed the hard-coded url string for the alias path.

Reply:
LOL, you're right.  I was overthinking this